solr/CHANGES.txt

```diff
@@ -317,6 +317,11 @@ Other Changes
 
 * SOLR-15385: Address many rawtypes warnings, resulting in several modified signatures in the public API. (Mike Drob, David Smiley, Christine Poerschke)
 
+* SOLR-15480: Added a new putAll methods for Tuple.fields to take care of marking EOF and EXCEPTION when those tokens are found.
+  The copy constructor (taking only fields), the merge, and the clone methods were updated to use the new putAll method. Additionally,
+  a new copy constructor taking another Tuple object was added and it, along with clone and merge, were refactored to handle setting
+  fieldNames and fieldLabels if necessary. (John Durham)
+
```

Comment:
3 of 3: Assuming we're targetting the `8.10.0` release (I think we are, @madrob do you agree?) let's relocate this into its section (ca. line 350 onwards below) and categorise it under "Bug Fixes" rather "Other Changes" section.

The description also seems rather long, how about something shorter e.g. _"Make Tuple copy constructor, clone and merge consistent w.r.t. markers (EOF, EXCEPTION), field names and labels. (John Durham via Mike Drob, Christine Poerschke)"_ i.e. focus on the outcome and omission of the implementation details?

Reply:
I think that makes sense. It shouldn't be about what was done specifically, but about what the result of the changes are.